### PR TITLE
Add categories to edit form

### PR DESCRIPTION
**Pull request by Gw3i**

### CODE REVIEW

File: src/components/EditFrom/EditForm.jsx [hunk 1]
```diff
@@ -89,6 +92,31 @@ export default function EditForm() {
...
+						</option>
+					))}
+				</select>
+				<ErrorMessage
+					errors={errors}
+					name="category"
+					render={({ messages }) =>
+						messages &&
+						Object.entries(messages).map(([type, message]) => (
+							<StyledInputWarning key={type} role="alert">
```
Comment (macfrei) [inline src/components/EditFrom/EditForm.jsx:113]: Is the `type` you are using for the `key` always unique?

**Review by macfrei — APPROVED** [review]
Good job, just one question!

File: src/components/EditFrom/EditForm.jsx (hunk 1 above)
Comment (Gw3i) [reply src/components/EditFrom/EditForm.jsx:113]: Yes, should be. Its preset from React Hook Form

Comment (macfrei) [reply src/components/EditFrom/EditForm.jsx:113]: Ahh good, thanks :)

**Pull request merged**